Content.Server/Pinpointer/PinpointerSystem.multiple.cs

```diff
@@ -0,0 +1,84 @@
+using Content.Shared.Interaction;
+using Content.Shared.Pinpointer;
+
+namespace Content.Server.Pinpointer;
+
+public sealed partial class PinpointerSystem
+{
+    public void InitializeMultiplePinpointer()
+    {
+        SubscribeLocalEvent<MultiplePinpointerComponent, ComponentStartup>(OnMultiplePinpointerStartup);
+        SubscribeLocalEvent<MultiplePinpointerComponent, ActivateInWorldEvent>(OnMultiplePinpointerActivated);
+        SubscribeLocalEvent<MultiplePinpointerComponent, AfterAutoHandleStateEvent>(OnMultiplePinpointerHandleState);
+    }
+
+    private void OnMultiplePinpointerHandleState(EntityUid uid, MultiplePinpointerComponent component, ref AfterAutoHandleStateEvent args)
+    {
+        SetMultiplePinpointer(uid, component);
+    }
+
+    private void OnMultiplePinpointerStartup(EntityUid uid, MultiplePinpointerComponent multiple, ComponentStartup args)
+    {
+        if (EntityManager.TryGetComponent(uid, out PinpointerComponent? tool))
```

Comment:
This should probably just be an `EnsureComp` rather than not doing anything if it isn't found.